### DISCUSSION TITLE
I want to install the extension AvaloniaVS.VS2022.vsix without an internet connection. Is there an offline installation package available? Where can I download it?

### DISCUSSION THREAD

**Discussion by qq908382818** [AvaloniaUI/Avalonia.Templates#253] [General]
I want to install the extension AvaloniaVS.VS2022.vsix without an internet connection. Is there an offline installation package available? Where can I download it?

**Comment by Mrxx99** [comment]
You can download the vsix from here https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.AvaloniaVS and then install it offline.
There is also the template studio extension here: https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.TemplateStudioForAvalonia

Btw this is not related to avalonia dotnet new templates as this repo is only about the CLI templates (that also show up in Visual Studio) and not about any VSIX. A better place for such questions would be the discussions section on the VS extension (https://github.com/AvaloniaUI/AvaloniaVS)
